nibabel/testing/__init__.py

```diff
@@ -7,23 +7,19 @@
 #
 ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
 ''' Utilities for testing '''
+from __future__ import division, print_function
+
+import warnings
 from os.path import dirname, abspath, join as pjoin
 
 import numpy as np
-from warnings import catch_warnings, simplefilter
+
+from nose.tools import (assert_equal, assert_not_equal,
+                        assert_true, assert_false, assert_raises)
 
 # set path to example data
 data_path = abspath(pjoin(dirname(__file__), '..', 'tests', 'data'))
 
-# Allow failed import of nose if not now running tests
-try:
-    import nose.tools as nt
-except ImportError:
-    pass
-else:
-    from nose.tools import (assert_equal, assert_not_equal,
-                            assert_true, assert_false, assert_raises)
-
```

Comment:
This is so that people can still import nibabel if they don't have nose installed.